packages/trader/src/AppV2/Components/BottomNav/bottomNav.scss

```diff
@@ -0,0 +1,33 @@
+.bottom-nav {
+    &-container {
+        display: flex;
+        padding: 8px;
+        align-items: flex-start;
+        gap: 10px;
+        align-self: stretch;
```

Comment:
`align-self` won't work with flex-container (only with flex items), so it can be deleted without any consequences I guess